code/game/objects/items/crayons.dm

```diff
@@ -730,13 +732,18 @@
 
 		return
 
-	if(istype(target, /obj/structure/window))
+	if(isobj(target) && !(target.flags_1 & UNPAINTABLE_1))
 		if(actually_paints)
+			var/list/rgb = hex2rgb(paint_color)
+			var/list/hsl = rgb2hsl(rgb[1], rgb[2], rgb[3])
+			var/color_is_dark = hsl[3] < DARK_COLOR_LIGHTNESS_THRESHOLD
```

Comment:
Luminosity threshold?

Reply:
Lightness and luminosity are different. After Googling for about ten minutes I still have no idea what the difference *is*, but I know they're different *somehow* and we use lightness.

Reply:
I cannot find the difference either, that works then